### DISCUSSION TITLE
How to set the screen resolution of macOS

### DISCUSSION THREAD

**Discussion by ctamietto** [quickemu-project/quickemu#1177] [Quickemu] [answered]
can I set the screen resolution of a mac sonoma ?

thanks for any answer

**Comment by flexiondotorg** [comment]
This feature is being worked on and tracked here: https://github.com/quickemu-project/quickemu/issues/1202

**Comment by flexiondotorg** [comment, marked as answer]
You will need Quickemu 4.9.5 or newer, which can set macOS display using the `--width` and `--height` command-line arguments.

To set the display resolution on macOS, you will also need to install a 3rd party resolution manager in the macOS VM to get the best results. We recommended using one of the following: 

- [Resolutionator](https://manytricks.com/resolutionator)
- [SwitchResX](https://www.madrau.com/)
- [BetterDisplay](https://github.com/waydabber/BetterDisplay) (*requires macOS 12.4 or newer*)

**Comment by Lazza** [comment]
I have seen excellent results with this one:

https://github.com/usr-sse2/RDM

    brew install --cask usr-sse2-rdm
